src/css/main.css

```diff
@@ -149,6 +149,37 @@ body,
   transform: translate3d(-50%, -50%, 0);
 }
 
+.progress-bar {
```

Comment:
These styles are a great start, to make the work in Firefox (and to be more in line with the rest of the app colorwise) refactor them to look something like:

```css
.progress-bar {
  grid-row-start: 4;
  grid-column-start: 1;
  grid-column-end: 6;
  border: none;
  background: #9c0629;
}

progress[value] {
  width: auto;
  height: 20px;
  -webkit-appearance: none;
}

progress[value]::-webkit-progress-bar {
  background-color: #9c0629;
}

progress[value]::-moz-progress-bar {
  background-color: #438365;
}

progress[value]::-webkit-progress-value {
  background-color: #438365;
}
```

Reply:
Thanks for your comments :) I will take care as soon as possible.